VocaDbModel/Service/VideoServices/VideoServiceBilibili.cs

```diff
@@ -53,18 +54,19 @@ public VideoServiceBilibili()
 		public override async Task<VideoUrlParseResult> ParseByUrlAsync(string url, bool getTitle) {
 
 			var id = GetIdByUrl(url);
+			var aid = id.StartsWith("BV") ? BilibiliHelper.Decode(id).ToString() : id;

```

Comment:
We should definitely save both aid and bvid. Bvid can be added to BiliMetadata, same as Cid.